System.IO.Abstractions.TestingHelpers/IMockFileDataAccessor.cs

```diff
@@ -66,5 +66,6 @@ public interface IMockFileDataAccessor
         PathVerifier PathVerifier { get; }
 
         IFileSystem FileSystem { get; }
+        string FixPath(string path, bool checkCaps = false);
```

Comment:
Note: Not sure if I should use  checpCaps (currently not), and if not, not sure if I should expose checkCaps to the interface

Reply:
I'd rather not add this somehow unrelated method here and instead try to delegate expanding paths to `PathBase`/`IPath`, see comment below.

Reply:
yes, I was looking for that! Fixed!